packages/core/contracts-ovm/insured-bridge/implementation/BridgeRouter.sol

```diff
@@ -157,11 +157,11 @@ contract BridgeRouter is OVM_CrossDomainEnabled {
         whitelistedToken.bridgePool = _bridgePool;
         // We want to prevent any situation where a token mapping is whitelisted on this contract but not on the
         // corresponding L2 contract.
-        require(depositContract != address(0), "Deposit contract not set");
+        require(depositContract != address(0), "Deposit contract not set"); //TODO: turn this into a modifier
```

Comment:
Can you remove changes to this file and just comment on the corresponding line in #3264 so as to avoid merge conflicts? Note that this contract is now named `BridgePoolFactory`

Reply:
done